tests/unit/memefficiency.tcl

```diff
@@ -51,6 +51,8 @@ run_solo {defrag} {
 
             populate 700000 asdf1 150
             populate 170000 asdf2 300
+            # Some volatile keys
+            populate 10000 asdf3 10 0 false 1000
```

Comment:
this could maybe cause issues for the fragmentation thresholds of this test.
any reason not to greatly reduce the number of keys?
i.e. as soon as we have some volatile keys (e.g. 1), that code path is covered (even if we don't re-locate them).
alternatively, if we wanna check that when we relocate a key name, the other dict is updated, i think we should set some TTLS to a few keys in the `asdf1` group (the one that suffers from eviction)
i.e.
```
r expire asdf11 1000
r expire asdf1799999 1000
```
or maybe a loop to set it to the first 100 and last 100 keys in each range?
i don't want more since i'm afraid it'll de-stabilize the fragmentation thresholds of the test.
WDYT?

Reply:
Added very small set of volatile keys to the existing keys `asdf1*` and `asdf2*`. It should ideally undergo some eviction code flow as well as scan through expire dictionary during defrag.